lib/widget/image.dart

```diff
@@ -135,6 +139,20 @@ class ImageState extends WidgetState<EnsembleImage> {
     return rtn;
   }
 
+  Future<String> fetch(String url) async {
+    final http.Response response = await http
+        .get(Uri.parse("${url}timeStamp=${DateTime.now().toString()}"));
```

Comment:
@fullstackctp how can this code ever work - you are putting timeStamp directly after the url without ny query string parameter. 

so if the url is myapp.com/img.png it will become myapp.com/img.pngtimeStamp=.... which will always fail.

You have to check if the url has a query string in it already. If it does, add a query parameter with &. If it doesn't add a new query string with ?

Was this code tested? What am I missing, this would never work

Reply:
also you mentioned that you stopped adding the current time but you are doing that here?